modin/pandas/test/test_io.py

```diff
@@ -696,17 +696,23 @@ def test_read_csv_quoting(
         )
 
     # Error Handling parameters tests
-    @pytest.mark.xfail(
-        Engine.get() != "Python",
-        reason="read_csv with Ray engine doen't raise `bad lines` exceptions - issue #2500",
-    )
     @pytest.mark.parametrize("warn_bad_lines", [True, False])
     @pytest.mark.parametrize("error_bad_lines", [True, False])
     def test_read_csv_error_handling(
         self,
+        request,
         warn_bad_lines,
         error_bad_lines,
     ):
+        if (
+            Engine.get() != "Python"
+            and request.config.getoption("--simulate-cloud").lower() != "normal"
+        ):
+            pytest.xfail(
+                "read_csv with Ray engine doesn't \
+                raise `bad lines` exceptions - issue #2500"
```

Comment:
this seems to be fixing something entirely different, if that is so pls not squeeze it in unrelated PR

Reply:
Without these changes the test failed as xpass.

Reply:
Can we check backend here (it would be `Cloudray` or `Cloudpython`) instead? this check is going to misbehave on `--simulate-cloud=experimental` I think, plus I don't like the idea of checking some test runner specific option - what if we rename it in `conftest.py` or change its values, but forget to change it here?